src/js/plugins/plugin.mobile_nav.js

```diff
@@ -133,50 +134,74 @@ BookReader.prototype.buildToolbarElement = (function (super_) {
 BookReader.prototype.buildMobileDrawerElement = function() {
   let experimentalHtml = '';
   if (this.enableExperimentalControls) {
-    experimentalHtml += "<div class=\"DrawerSettingsTitle\">Experimental (may not work)</div>"
-        +"        <button class='BRaction default high-contrast-button'>Toggle high contrast</button>";
+    experimentalHtml = `
+        <p class="DrawerSettingsTitle">Experimental (may not work)</p>
```

Comment:
I'm a little wary of changing the HTML itself; that might have subtle regressions in how things look which would be kind of annoying to test/deal with.

```suggestion
        <div class="DrawerSettingsTitle">Experimental (may not work)</div>
```

Reply:
hmmm, even after I checked it? p offers the same `block` display + it's semantic. It displays as expected

Reply:
+ it has a class to attach a style to.  I will hold with the update, it returns as the same on prod and demo